FEM/rf_num_new.cpp

```diff
@@ -116,7 +116,7 @@ CNumerics::CNumerics(string name)
 		cpl_error_tolerance[i] = -1.0; // JT2012: should not default this. Should always be entered by user!
 	//
 	// ELE
-	ele_gauss_points = 3;
+	ele_gauss_points = 2;
```

Comment:
why did you change the default number of gauss points? This should influence lots of benchmarks.